### ISSUE TITLE
Audio not working in Example Generator

### ISSUE THREAD

**Issue by olets** [paulrosen/abcjs#770]
Audio is broken in the Example Generator https://paulrosen.github.io/abcjs/overview/example-generator.html.

Result after clicking "activate audio"

<img width="660" alt="image" src="https://user-images.githubusercontent.com/3282350/159192756-f03a2106-c2b5-4a8e-8a1c-8c7600b95834.png">

There's a console error

> abcjs-audio.css:1          Failed to load resource: the server responded with a status of 404 ()

Looking at the markup suggests that that's one problem:

```html
<div class="abcjs-css-warning" style="font-size: 12px;color:red;border: 1px solid red;text-align: center;width: 300px;margin-top: 4px;font-weight: bold;border-radius: 4px;">CSS required: load abcjs-audio.css</div>
```

But in a local React sandbox I have the css loaded, and get the same result when using the Example Generator's generated code (no error in the console, but grey bar on the front end and a CSS error message discoverable in the markup).

```jsx
import abcjs from "abcjs";
import "abcjs/abcjs-audio.css";

function Abc() {
  const abcString = 'X:1\nT:Some Tune\nK:G\nAAAA AAAA |\n';

  const visualOptions = {};

  const visualObj = abcjs.renderAbc("*", abcString, visualOptions);

  activate() {
    if (abcjs.synth.supportsAudio()) {
        const controlOptions = {
            restart: true,
            play: true,
            progress: true,
            clock: true
        };

        const synthControl = new abcjs.synth.SynthController();

        synthControl.load("#audio", null, controlOptions);

        synthControl.disable(true);

        const midiBuffer = new abcjs.synth.CreateSynth();

        midiBuffer.init({ visualObj: visualObj[0] })
          .then(function () {});
    } else {
        console.log("audio is not supported on this browser");
    };
  }

  return (
    <div>
      <div id="audio"></div>
      <button className="activate-audio" onClick={activate}>Activate Audio</button>
    </div>
  )
}

export default Abc

**Comment by paulrosen** [comment]
Will be fixed in v6.0.2, out later today.

**Comment by olets** [comment]
It works! Thank you!
